dynamics/apis/dimension_values.py

```diff
@@ -0,0 +1,20 @@
+from .api_base import ApiBase
+
+
+class DimensionValues(ApiBase):
+    """Class for Locations APIs."""
+
+    GET_DIMENSION_VALUES = '/dimensions({0})/dimensionValues'
```

Comment:
can we merge all dimensions into one file

have get_all, get_all_values and post_dimensions  functions

Add support for count as well for diemensions and dimension values